FloconDesktop/navigation/src/commonMain/kotlin/io/github/openflocon/navigation/FloconNavigationState.kt

```diff
@@ -0,0 +1,16 @@
+package io.github.openflocon.navigation
+
+class FloconNavigationState internal constructor() {
```

Comment:
please : 

1. split into interface/impl
```
interface FloconNavigationState {
    val stack: State<List<FloconRoute>>

    fun navigate(route: FloconRoute)

    fun back(count: Int = 1)
}
```

2. use a mutable state & collect 
// not sure mutableStateListOf exists, maybe mutableStateOf<List<FloconRoute>>(
```
class FloconNavigationState internal constructor() {

    private val _stack = mutableStateListOf<FloconRoute>(LoadingRoute)
    val stack: State<List<FloconRoute>> = _stack

    fun navigate(route: FloconRoute) {
        _stack.add(route)
    }

    fun back(count: Int = 1) {
        repeat(count) { _stack.removeLast() }
    }

}
```

```
val stack by navigationState.stack
```